tests/functional/adapter/utils/test_timestamps.py

```diff
@@ -0,0 +1,20 @@
+import pytest
+from dbt.tests.adapter.utils import test_timestamps
+
+
+class TestCurrentTimestampSnowflake(test_timestamps.TestCurrentTimestamps):
```

Comment:
Probably want `TestCurrentTimestampRedshift` rather than `TestCurrentTimestampSnowflake`.

Side note:
- I don't know why we have a convention for including the data platform name at the end of the class name! 🤷 
- It seems like it would make copy-paste a lot easier if we just did `class TestCurrentTimestamp(`... instead. We can bring it up with the group and see if there's a reason for the platform postfix.

Otherwise, same commentary as [here](https://github.com/dbt-labs/dbt-snowflake/pull/273/files#r984113666).